packages/nimble-components/src/select/tests/select.spec.ts

```diff
@@ -738,6 +738,22 @@ describe('Select', () => {
         await disconnect();
     });
 
+    it('when selected option textContent is updated directly in DOM, display text is updated', async () => {
```

Comment:
Can you write one more test that verifies that this functionality works correctly if the display value wasn't directly text content (i.e. in a span instead)?

Reply:
Note that this required setting `childList: true` in the options for the mutation observer.